website/package.json

```diff
@@ -70,5 +74,6 @@
   "peerDependencies": {
     "react": "18.x",
     "react-dom": "18.x"
-  }
+  },
+  "packageManager": "yarn@4.4.0"
```

Comment:
I had issues mixing volta and packageManager. I personally prefer volta as it makes it way easier to move between repos and directories.

Reply:
It is possible to add a volta section that extends the base package.json, but I got some trouble when adding website to workspaces, so keep a separate volta section in website. Just my experience...